docs/guides/primitives.mdx

```diff
@@ -6,6 +6,26 @@ description: Introduction to primitives in Qiskit and Qiskit Runtime, and an exp
 
 # Introduction to primitives
 
+## Why does Qiskit use primitives?
```

Comment:
It's more accurate to say Qiskit _introduced_ primitives, instead of merely using them, since the base interfaces live in Qiskit.